tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md

```diff
@@ -17,12 +17,21 @@ author_profile: https://github.com/dhrubpaul
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](API to list available scenarios)]
+[ACCORDION-BEGIN [Step 1: ](List scenarios)]
 
-Scenarios refers to the use case. It is created by an identifier within the workflows *(YAML files, see screenshots below)* that you synced with GitHub in previous tutorial. For all workflows of same use case, write the same scenario id.
+Scenarios refers to an ML use case. It gets automatically created when you upload workflows *(YAML files, see screenshots below)* to your GitHub repository connected with your SAP AI Core.
+
+Following are the annotations within your training workflow `training_workflow_tutorial.yaml` that define a scenario.
```

Comment:
Please fix grammar. It is only one workflow.

Reply:
I assume you are referring to "created when you upload workflows" which should then read "created when you upload a workflow" or "created when you upload the workflows"

Reply:
done